### DISCUSSION TITLE
What's the right way to access "TestingT: t," from inside the "Step" function?

### DISCUSSION THREAD

**Discussion by alexanderilyin** [cucumber/godog#652] [Q&A]
I'm trying to use [`humatest`](https://github.com/danielgtaylor/huma/tree/main/humatest) and `godog` at the same time. Currently I'm doing this:

```go
package main
// ...
var test *testing.T
// ...
func iSendPOSTRequestTo(ctx context.Context, url string) error {
	_, api := humatest.New(test)
        // ...
}
```
but it feels wrong.

**Comment by Johnlon** [comment]
Doesn't godog.T(ctx) return godog's wrapper testingT ?
Ie testingT  wraps testing.T
